[skip-ci] Add distributed RDF paragraph to RelNotes. (#18686)
root-project/root: README/ReleaseNotes/v636/index.md

### Distributed RDataFrame

The distributed RDataFrame (`ROOT.RDF.Distributed`) is now out of the experimental namespace. We recommend that users switch to using the new version as soon as possible. Note the following changes between the old and the new code:

Using dask:
```python
# old version - not recommended: 
# this will still work in 6.36 but will raise a warning
df = ROOT.RDF.Experimental.Distributed.Dask.RDataFrame

# new version - recommended: 
df = ROOT.RDataFrame("mytree", "myfile.root", executor=daskclient
```

Using spark: 
```python
# old version - not recommended: 
# this will still work in 6.36 but will raise a warning
df = ROOT.RDF.Experimental.Distributed.Spark.RDataFrame

# new version - recommended: 
df = ROOT.RDataFrame("mytree", "myfile.root", executor=sparkcontext)
```

> In case of using the old version, with `Experimental` keyword, the warning will be raised. Possibility of using this old version will be completely removed in ROOT 6.38.

Reviewer: ```suggestion
In case of using the old version, with `Experimental` keyword, a warning will be raised. In ROOT 6.38, the `Experimental` name will be removed completely and using it will result in an exception.
```
